include/aoapplication.h

```diff
@@ -433,6 +441,14 @@ class AOApplication : public QApplication {
   // The file name of the log file in base/logs.
   QString log_filename;
 
+  void initBASS();
```

Comment:
why is initbass not inside the bassaudio ifdef

Reply:
its just the function definition, shouldn't matter too much